integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -153,6 +153,20 @@ def _transmit_action(self, content_metadata_item_map, client_method, action_name
         for chunk in islice(chunk_items, transmission_limit):
             json_payloads = [item.channel_metadata for item in list(chunk.values())]
             serialized_chunk = self._serialize_items(json_payloads)
+            if self.enterprise_configuration.dry_run_mode_enabled:
+                encoded_serialized_payload4 = encode_binary_data_for_logging(serialized_chunk)
```

Comment:
i wrote the encoded data logging facility and i am undecided if it is actually useful.

you may need to/want to log each item, not a whole chunk (which is often ~500 courses). this is because splunk has a line-length limit and will truncate long log lines. if you chose to use the encoded data logging you likely need to log each item in the chunk.

Reply:
Hi @johnnagro, should I encode the data while logging? Is there any specific use case we have due to which we are encoding this data while logging?

Reply:
Not sure about encoding, but I second @johnnagro's point on iterating over and logging individual items in the chunk. We've run into situations where critical pieces of logs/identifiers were lost because the log was truncated.